frappe_pwa/frappe_pwa/doctype/web_app_manifest/test_web_app_manifest.py

```diff
@@ -6,5 +6,13 @@
 import frappe
 import unittest
 
+from frappe_pwa.frappe_pwa.doctype.web_app_manifest.web_app_manifest import configure_pwa
+
+
 class TestWebAppManifest(unittest.TestCase):
-	pass
+    def test_configure_pwa(self):
+        configure_pwa()
+        ws = frappe.get_doc('Website Settings')
+        self.assertIn('<link href="/assets/frappe_pwa/manifest.json" rel="manifest">', ws.head_html)
+        ws.head_html.replace('<link href="/assets/frappe_pwa/manifest.json" rel="manifest">', '')
+        ws.save()
```

Comment:
I understand you want to "_restore_" the Settings to their initial value, but :
1. You should do this too before calling `configure_pwa()` to ensure the function actually does what's expected
2. Those should be in the setup and tearDown functions dedicated to prepare the tests, not in the test itself
3. Is this really necessary ?